src/codeExamples/addIcon.stories.mdx

```diff
@@ -0,0 +1,36 @@
+import { Meta } from '@storybook/addon-docs';
+import copySVGcode from './images/CopySVGcode.png';
+
+<Meta title="code examples/add svg icon to DS icon library" />
+
+# Add SVG to the Design System icon library
+
+When you need to add a new icon that is going to be used inside of the design system library, use the following step-by-step guide.
+
+## What doesn't belong in the DS icon library
+
+The design system library only has icons that are free license only and are used within the DS components as preset icons. If the icon can be added to the component through the component API, it doesn't need to be included in the design system library.
+
+## Step-by-step guide
```

Comment:
Would be nice to mention that if FA icon is added it should contain also the license comment